.env.example

```diff
@@ -1,9 +0,0 @@
-SECRET_KEY_BASE=
```

Comment:
We are going to have a separate config file for the elastic beanstalk to deploy the app and we don't need env file for dev environment.